contrib/kind.sh

```diff
@@ -788,26 +800,19 @@ build_ovn_image() {
   if [ "$OVN_IMAGE" == local ]; then
     set_ovn_image
 
-    # Build ovn image
-    pushd ${DIR}/../go-controller
-    make
-    popd
+    # Build binaries
+    make -C ${DIR}/../go-controller
 
-    # Build ovn kube image
-    pushd ${DIR}/../dist/images
-    # Find all built executables, but ignore the 'windows' directory if it exists
-    find ../../go-controller/_output/go/bin/ -maxdepth 1 -type f -exec cp -f {} . \;
-    echo "ref: $(git rev-parse  --symbolic-full-name HEAD)  commit: $(git rev-parse  HEAD)" > git_info
-    $OCI_BIN build -t "${OVN_IMAGE}" -f Dockerfile.fedora .
+    # Build image
+    make -C ${DIR}/../dist/images IMAGE="${OVN_IMAGE}" OVN_REPO="${OVN_REPO}" OVN_GITREF="${OVN_GITREF}" fedora-image
```

Comment:
@qinqon you broke podman users here when you've dropped the $OCI_BIN variable.

Please review https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5065 

@jcaamano for (hopefully ...) a quick merge ...

Reply:
Bummer, will review ASAP, sorry.